cartoframes/client/client_factory.py

```diff
@@ -0,0 +1,5 @@
+from .api_client import APIClient
+
+
+def get_client(creds, session=None):
```

Comment:
I would rename this to `create_client` and move to the `client/__init__.py` file

Reply:
:+1: